test/e2e/tests/simple-send.spec.js

```diff
@@ -14,14 +19,28 @@ describe('Simple send', function () {
         ganacheOptions: defaultGanacheOptions,
         title: this.test.fullTitle(),
       },
-      async ({ driver, ganacheServer }) => {
-        await logInWithBalanceValidation(driver, ganacheServer);
+      async ({ driver }) => {
+        // Process
+        await unlockWallet(driver, WALLET_PASSWORD);
 
-        await sendTransaction(
```

Comment:
this method is at the moment defined in `helpers` file and it has a flavor of a process - understood in the POM context. The decision here was to, instead of directly implement a process for send-transaction (which we will very likely need) just to implement the steps one by one in the test.
When to add a process? if this group of steps is repeated in another tests then, we could abstract it as a process (like done in unlock wallet above).

This approach serves a demo too, as we can see here both processes and normal page object actions using their methods in the specs